gymnasium/utils/ezpickle.py

```diff
@@ -19,17 +19,15 @@ class EzPickle:
 
     def __init__(self, *args, **kwargs):
         """Uses the ``args`` and ``kwargs`` from the object's constructor for pickling."""
-        self._ezpickle_args = args
```

Comment:
I removed them with reasons outlined in https://github.com/WillDudley/Gymnasium/pull/2

But perhaps I'm missing something regarding their importance/usage.

Reply:
Following discussions on Discord, I've added back ezpickle_args for legacy purposes - they will not work with spec_stack functionality but will raise a warning  when detected and the "old way" of using ezpickle will still work (nothing in this PR would break it).